.github/workflows/deploy-realm.yml

```diff
@@ -0,0 +1,36 @@
+name: realm
+
+on:
+  push:
+    paths:
+      - "realm/**"
+    branches:
+      - main
+
+env:
+  GNOKEY: "go run github.com/gnolang/gno/gno.land/cmd/gnokey"
+  GNOCHESS_REALM: gno.land/r/demo/chess_${GITHUB_SHA}
```

Comment:
would be nice to deploy based on github_pr once merged too.

Reply:
hum, googled and can't find an easy way of getting the PR number of a merged commit, only when merging on master/main. :/